docs/configuration.md

```diff
@@ -474,6 +474,22 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.shuffle.service.enabled</code></td>
+  <td>false</td>
+  <td>
+    Enables the external shuffle service. This service preserves the shuffle files written by 
+    executors so the executors can be safely removed. This must be enabled if 
```

Comment:
Pardon my ignorance -- i don't pay much attention to this part -- but what does this entail? like, is there a bit of doc I can point to here?